src-java/network-topology/network-storm-topology/src/main/java/org/openkilda/wfm/topology/network/storm/bolt/isl/IslHandler.java

```diff
@@ -207,6 +241,21 @@ public void auxiliaryPollModeUpdateRequest(Endpoint endpoint, boolean enableAuxi
         emit(STREAM_POLL_ID, makePollTuple(command));
     }
 
+    @Override
+    public void processIslRuleInstalled(IslReference reference, Endpoint endpoint) {
```

Comment:
Move these methos down, to all other `process*` methods. They are grouped for a reason. They are entry points for stream commands.

Reply:
Renamed.